src/main/java/com/juniorstart/juniorstart/model/dto/ProjectDto.java

```diff
@@ -0,0 +1,18 @@
+package com.juniorstart.juniorstart.model.dto;
+
+import com.juniorstart.juniorstart.model.User;
+import com.juniorstart.juniorstart.model.audit.ProjectStatus;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class ProjectDto {
+
+    private long id;
+    private String title;
+    private ProjectStatus progress;
+    private String description;
+    private String url;
+    private User user;
+}
```

Comment:
Ja bym to chyba zrobił jako statyczną klasę w klasie Project. Statyczna, żeby nie trzeba było tworzyć obiektu Project do stworzenia ProjectDto.